ui/view/controls/OpenInBlockchainExplorer.qml

```diff
@@ -12,17 +12,6 @@ Item {
     Layout.preferredWidth: (control.showText ? openInExplorer.width + 10 : 10) + openInExplorerIcon.width
     height: 16
 
-    SFText {
```

Comment:
why deleted? "showText = false" will disable this

Reply:
Fixed.